docs/ref/contrib/postgres/search.txt

```diff
@@ -261,6 +266,23 @@ floats to :class:`SearchRank` as ``weights`` in the same order above::
     >>> rank = SearchRank(vector, query, weights=[0.2, 0.4, 0.6, 0.8])
     >>> Entry.objects.annotate(rank=rank).filter(rank__gte=0.3).order_by('-rank')
 
+Lexeme objects
+==============
+
+Lexeme objects allow search operators to be used along with strings from an untrusted source. The contents
+of each lexeme is excaped so cannot contain any operators that PostgreSQL will recognise, but they support
```

Comment:
"escaped"

Reply:
also plural "the contents of each lexeme _are_ escaped"
and "so cannot contain" is not clear, sounds like it's missing a word.

Reply:
"recognize" for American English, don't blame me, it's CI that will complain :-)

Reply:
Thanks! I think changing "contents" to singular sounds a bit better, but I'm not sure!

"the content of each lexeme is escaped"

Reply:
I've had a go at rewriting this paragraph, let me know what you think! I'm not sure about the wording of first sentence in particular